PodcastGenerator/core/episodes.php

```diff
@@ -31,15 +31,20 @@ function getEpisodes($category = null, $_config)
             $this_mtime = filemtime($this_entry);
             if (in_array(pathinfo($this_entry, PATHINFO_EXTENSION), $supported_extensions)
                 && file_exists($_config['absoluteurl'] . $_config['upload_dir'] . pathinfo($this_entry, PATHINFO_FILENAME) . '.xml')
-                && $this_mtime <= $now_time) {
+                && ($this_mtime <= $now_time || isset($_SESSION["username"]))) {
```

Comment:
Please put the username in single quotes, it should be the same syntax per file

Reply:
Fixed